### PR TITLE
Fix/2005 handle exploded form query params

### PR DESCRIPTION
**Pull request by chohmann**
Addresses #2005 

**Summary**

Handle exploded form query params for types:
- string
- array
- object

Had to update the colorizer code to not attempt to add the prefix and postfix for objects, since that was what was breaking this in the first place.

Also updated the default for form query params to be exploded true to match the spec in the `createQueryUriTemplate` function.

Matches what the spec says: https://spec.openapis.org/oas/v3.0.3#style-examples

**Checklist**

- The basics
  - [x] I tested these changes manually in my local or dev environment
- Tests
  - [x] Added or updated
  - [ ] N/A
- Event Tracking
  - [ ] I added event tracking and followed the event tracking guidelines
  - [ ] N/A
- Error Reporting
  - [ ] I reported errors and followed the error reporting guidelines
  - [ ] N/A

**Screenshots**

### Non-Exploded form object param:
```parameters:
        - name: pagination
          in: query
          required: true
          style: form
          explode: false
          schema:
            type: object
            properties:
              offset:
                type: integer
                example: 0
              limit:
                type: integer
                example: 25
```
image

### Exploded form object param:
```
parameters:
        - name: pagination
          in: query
          required: true
          style: form
          explode: true
          schema:
            type: object
            properties:
              offset:
                type: integer
                example: 0
              limit:
                type: integer
                example: 25
```
image

### Non-Exploded form array param:
```
parameters:
        - name: pagination
          in: query
          required: true
          style: form
          explode: false
          schema:
            type: array
            items:
              type: string
```
image

### Exploded form array param:
```
parameters:
        - name: pagination
          in: query
          required: true
          style: form
          explode: true
          schema:
            type: array
            items:
              type: string
```
image

### Non-Exploded form string param:
```
parameters:
        - name: pagination
          in: query
          required: true
          style: form
          explode: false
          schema:
            type: string
```
image

### Exploded form string param:
```
parameters:
        - name: pagination
          in: query
          required: true
          style: form
          explode: true
          schema:
            type: string
```
image

### CODE REVIEW

File: test-harness/specs/validate-query-params/query-param-form-exploded-object-invalid.oas3.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+              limit:
+                type: integer
+                example: 25
+      responses:
+        '200':
+          description: OK
+====server====
+mock -p 4010 ${document}
+====command====
+curl -i -X GET http://127.0.0.1:4010/recipes\?limit\=string\&offset\=string
```
Comment (daniel-white) [inline test-harness/specs/validate-query-params/query-param-form-exploded-object-invalid.oas3.txt:32]: why escaping here and not other places?

**Review by daniel-white — APPROVED** [review]
just a question about shell excaping

File: test-harness/specs/validate-query-params/query-param-form-exploded-object-invalid.oas3.txt (hunk 1 above)
Comment (chohmann) [reply test-harness/specs/validate-query-params/query-param-form-exploded-object-invalid.oas3.txt:32]: I'm not entirely sure... without the escaping, I wasn't getting both parameter validations, I was only getting offset. And this is how my terminal handled it when I pasted the url in there, and so I copied the escaping over and it got both validations to come across in the test 🤷‍♀️

**Comment by chohmann** [comment]
I'm going to spend like 30 minutes trying to get the color around the object param values successfully. If it takes longer than that, I'm going to let it be.

**Comment by chohmann** [comment]
well that was easy [c746bd2](https://github.com/stoplightio/prism/pull/2288/commits/c746bd2304fd62eb75a0183c41318ebea71624b1) 😂

**Comment by chohmann** [comment]
so now the cyan color is returned for the generated param values:
image

image

**Pull request merged**